PubSub.Application/Controllers/SubscribeController.cs

```diff
@@ -29,9 +28,31 @@ public SubscribeController(ISubscriberService subscriberService, ISubscribeReque
     public async Task<IActionResult> Subscribe([FromBody] SubscribeRequestDto request)
     {
         if (!ModelState.IsValid) return BadRequest();
-        var subscription = _subscribeRequestMapper.FromDto(request);
-        var subscriptions = new List<Subscription>() { subscription };
-        await _subscriberService.AddSubscriptionsAsync(subscriptions);
+        var subscriptionCreationRequests = _subscriptionMapper.FromDTO(request);
+        await _subscriptionService.AddSubscriptionsAsync(subscriptionCreationRequests);
         return NoContent();
     }
+
+    [HttpDelete]
+    [Route("{uuid:guid}")]
+    [ProducesResponseType(StatusCodes.Status204NoContent)]
+    [ProducesResponseType(StatusCodes.Status400BadRequest)]
+    [ProducesResponseType(StatusCodes.Status401Unauthorized)]
+    [ProducesResponseType(StatusCodes.Status403Forbidden)]
+    [ProducesResponseType(StatusCodes.Status404NotFound)]
+    public async Task<IActionResult> Delete(Guid uuid)
+    {
+        var result = await _subscriptionService.DeleteSubscription(uuid);
+        return result.Match(FromOperationError, NoContent);
+    }
+
+    [HttpGet]
+    [ProducesResponseType(typeof(IEnumerable<SubscriptionResponseDTO>), StatusCodes.Status200OK)]
+    [ProducesResponseType(StatusCodes.Status401Unauthorized)]
+    public async Task<IActionResult> GetSubscriptions()
```

Comment:
Maybe change the name to reflect the service - so it's only active user subscriptions